Sources/SwiftDocC/Semantics/Options/AutomaticArticleSubheading.swift

```diff
@@ -19,22 +19,21 @@ import Markdown
 public class AutomaticArticleSubheading: Semantic, AutomaticDirectiveConvertible {
     public let originalMarkup: BlockDirective
     
-    /// The specified behavior for automatic subheading generation.
+    /// Whether or not DocC generates automatic second-level "Overview" subheadings. By default it does.
     @DirectiveArgumentWrapped(name: .unnamed)
-    public private(set) var behavior: Behavior
+    public private(set) var enabledness: Enabledness
```

Comment:
I can definitely be convinced that `Enabledness` is the best word here – I'm mostly worried about it from a documentation perspective for folks less familiar with English... Just to offer an alternative – what do you think of `state: State`? It's in a state of enabled or disabled.

Reply:
I also couldn't think a good word here. What I really want it to describe is a Bool value that can be created with "enabled"/"disabled" instead of "true"/"false" so that the directive can be written as `@Directive(disabled)` instead of `@Directive(enabled: false)`.

After thinking about this some more I decided to add a new initializer to the property wrapper that allows for a Bool value with custom true/false spellings. That way we don't need to define a custom type for it however we also don't get to document the meaning of the values but it should be clear enough with a `var enabled: Bool` property what the two values are.

Reply:
Would we expect the declaration in documentation for this to be:

```
@Directive(_ enabled: Bool)
```

then?

I'm worried that will be more confusing since other instances of `Bool` in declarations accept `true` `false` as arguments.

Reply:
would it help if this also supported true and false as arguments?

Reply:
I don't think so... I'm just more concerned about the user-facing Directive documentation than the API framework documentation. For the Directive itself – using something like `Enabledness` or `State` is probably better than `Bool` since it indicates that this is a different thing than: https://ethan-kusters.github.io/swift-docc/documentation/docc/choice

Reply:
We can definitely land this as-is though and tweak the documentation in a future PR if you think this is the right spelling for the internal API.

Reply:
I'm not sure what the right internal API is for this. I think that the code should expose a `Bool` value—but that could be done with a computed property—but the user-facing Directive documentation should show something more specific than `Bool`. 

For the type name to surface in the directive documentation I prefer `Enabledness` over `State`. To me "state" implies the computed runtime value (which may change) as opposed to a user configuration. Also, "state" is unnecessarily broad and could have more states than enabled/disabled. Even if it's the first time the developer sees the word "enabledness" it should be fairly easy to discard what it is and what it's possible values are.

Reply:
I'll revert back to an `Enabledness` enum but will surface a computed Bool value to the internal API.

Reply:
That sounds good to me. Thank you!

Reply:
Ok. I've updated this to use an "enabledness" enum again and have regenerated the directives symbol graph file.

<img width="944" alt="Screenshot 2023-01-17 at 4 11 37 PM" src="https://user-images.githubusercontent.com/1228143/213043277-bb6cb360-fe7b-4461-9a2f-1b5764d25493.png">

Reply:
Looks great! Thank you.